MovieMVVM/MovieMVVMTests/AssemblyBuilderTest.swift

```diff
@@ -0,0 +1,23 @@
+// AssemblyBuilderTest.swift
+// Copyright © Ilentiy. All rights reserved.
+
+@testable import MovieMVVM
+import XCTest
+
+final class AssemblyBuilderTest: XCTestCase {
+    private var builder: AssemblyBuilderProtocol?
+
+    override func setUp() {
```

Comment:
```suggestion
    override func setUp() {
```
super везде добавь по файлам